### PR TITLE
Fixed email validation

### PR DESCRIPTION
**Pull request by mimse**
Fixed e-mail validation

image

### CODE REVIEW

File: libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts [hunk 1]
```diff
@@ -96,7 +96,10 @@ export class DhInviteUserModalComponent implements AfterViewInit, OnDestroy {
     actorId: ['', Validators.required],
     firstname: ['', Validators.required],
     lastname: ['', Validators.required],
-    email: [{ value: '', disabled: true }, Validators.required],
+    email: [
+      { value: '', disabled: true },
+      [Validators.required, Validators.pattern('^[a-zA-Z0-9._%+-]+')],
```
Comment (dzhavat) [inline libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts:101]: suggestion: the pattern and translations are out of sync. There are more symbols in the pattern than the translation mentions. Should we synchronize them?

Comment (mimse) [reply libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts:101]: I just wrote what @PernilleChristiansen suggested, and allowed for all valid chars

Comment (dzhavat) [reply libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts:101]: Can it be something like 'Only write letters, numbers, dots, and "_, %, +, -"'?

Comment (mimse) [reply libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts:101]: Changed to "Type a valid email"

**Review by dzhavat — APPROVED** [review]

**Pull request merged**